web/client/localConfig.json

```diff
@@ -30,6 +30,13 @@
     "monitorState": [
       {"name": "routing", "path": "routing.location.pathname"},
       {"name": "featuregridmode", "path": "featuregrid.mode"}],
+    "projectionDefs": [
```

Comment:
I won't add any projection by default (why EPSG:3003 and not EPSG:3035?)

Reply:
sure, this was an intermediate pr. i'll remove it.